datahub-web-react/src/app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/sources.json

```diff
@@ -0,0 +1,450 @@
+[
+    {
```

Comment:
would it be possible to reuse the original sources.json here? and just add the new items. i'd rather not have two if we can help it because both will be sitting around for a while and people will get confused on which one to add a new source to